tests/integration/questionnaire/test_questionnaire_custom_summary.py

```diff
@@ -0,0 +1,120 @@
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestCustomSectionSummary(IntegrationTestCase):
+    def add_person(self, first_name, last_name):
+        self.post({"anyone-else": "Yes"})
+
+        self.post({"first-name": first_name, "last-name": last_name})
+
+    def get_link(self, rowIndex, text):
+        selector = f"tbody:nth-child({rowIndex}) td:last-child a"
+        selected = self.getHtmlSoup().select(selector)
+
+        filtered = [html for html in selected if text in html.get_text()]
+
+        return filtered[0].get("href")
+
+    def get_previous_link(self):
+        selector = "#top-previous"
+        selected = self.getHtmlSoup().select(selector)
+        return selected[0].get("href")
+
+    def test_happy_path(self):
```

Comment:
Given how this test is named should there be a test for an unhappy path as well? Generally a bit confusing for me on what exactly is being tested here with removing a mistake etc.

Reply:
Thanks for the insight Rhys - I'll try to keep this in mind when naming tests in future. In any case this has now been removed 👍